packages/frontend/src/components/table/cells/type-info.tsx

```diff
@@ -78,9 +78,9 @@ export const providerMap: Record<
     Icon: ArbitrumIcon,
     text: "This project is based on Arbitrum's code base.",
   },
-  Polygon: {
+  'Agglayer CDK': {
     Icon: PolygonIcon,
-    text: "This project is based on Polygon's code base.",
+    text: "This project is based on Polygon's Agglayer CDK code base.",
```

Comment:
```suggestion
    text: "This project is based on the Agglayer CDK code base.",
```

my understanding is that Agglayer doesn't want to be associated with Polygon